services/visual_recognition/v3.js

```diff
@@ -350,6 +346,55 @@ module.exports = function (RED) {
     });
   }
 
+  // This function performs the operation to Delete ALL Dialogs
+  function performDeleteAllClassifiers(visualRecognition, node, msg) {
+
+    node.status({fill:'blue', shape:'dot', text:'requesting Delete All classifiers'});
+
+    var params = {};
+    visualRecognition.listClassifiers(params, function(err, body, other) {
+      node.status({});
+      if (err) {
+          node.status({fill:'red', shape:'ring', text:'Delete All : call to listClassifiers failed'});
+          node.error(err, msg);   
+        } else {
+          // Array to hold async tasks
+          var asyncTasks = [];
+          var nb_todelete = body.classifiers.length;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'nb_todelete' is not in camel case.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2712969112/issues/source?bid=3330694&fileBranchId=3344618#l363)